dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3530,10 +3504,58 @@ def test_options_ingested_correctly(self):
         self.assertFalse(self.profiler2.options.data_labeler.is_enabled)
 
 
-class TestStructuredProfilerNullValues(unittest.TestCase):
+class TestStructuredProfilerRowStatistics(unittest.TestCase):
     def setUp(self):
         test_utils.set_seed(0)
 
+    @classmethod
+    def setUpClass(cls):
+        test_utils.set_seed(seed=0)
+
+        data = [
+            "orange",
+            "green",
+            "blue",
+            "mexico",
+            "france",
+            "morocco",
+            "chevy",
+            "ford",
+            "toyota",
+            "apple",
+        ] * 10
+
+        for x in range(100):
+            data.append(f"{x}")
+
+        cls.data = pd.DataFrame(data)
+
+        profiler_options_hll = ProfilerOptions()
+        profiler_options_hll.set(
+            {
+                "*.is_enabled": False,
+                "row_statistics.*.is_enabled": True,
+                "row_statistics.unique_count.hashing_method": "hll",
```

Comment:
Why the `*` in `"row_statistics.*.is_enabled": True,`? Does it enable both row and below then?